pyproject.toml

```
@@ -13,13 +13,15 @@ authors = [
     { name = "Christian Amsüss", email = "chrysn@fsfe.org" },
     { name = "the aiocoap contributors" },
 ]
-license = { text = "MIT" }
+license-expression = "MIT AND BSD-3-Clause"
```

```suggestion
license = "MIT AND BSD-3-Clause"
```

Got it: `license = "foo"` is license expressions, whereas `license = { text = "foo" }` means the old and deprecated form.